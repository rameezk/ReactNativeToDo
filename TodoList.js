import React from 'react'
import { StyleSheet, View, Text } from 'react-native'

const TodoList = () => {
  const { container } = styles
  return (
    <View style={container}>
      <Text>This is a todo list</Text>
    </View>
  )
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
  },
})

export default TodoList
