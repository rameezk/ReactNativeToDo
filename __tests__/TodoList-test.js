import React from 'react'
import {} from 'react-native'

import TodoList from '../TodoList'

import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(<TodoList />).toJSON()
  expect(tree).toMatchSnapshot()
})
