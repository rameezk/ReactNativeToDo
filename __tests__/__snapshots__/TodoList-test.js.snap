// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
  >
    This is a todo list
  </Text>
</View>
`;
